Automattic/jetpack
# Update the styling and formatting of the form submission page

The styling of the page shown after a form is submitted is updated by this PR. It is a visual only change.

Fixes  #27295.

#### Changes proposed in this Pull Request:
* The styling of the form submission page is updated. The exact styling of fonts used will depend on styling of the theme.

pcsBup-py-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:
* Using this branch, create a contact form block in a post or page.
* Preview the page and fill in the survey and submit it.
* The page shown after submitting should look much like that in #27295

Before:

<img width="542" alt="200317711-c58ec9bb-a503-4a12-b42a-c7e467782482" src="https://user-images.githubusercontent.com/5656673/201707170-4033110e-26a9-4dfb-a8ba-4e1dd5782a49.png">

After:
![Screenshot 2022-11-14 at 16 00 48](https://user-images.githubusercontent.com/5656673/201707292-dab0897e-2c27-421e-a5bd-174f531e1899.jpg)
